test/e2e/seeder/ganache-seeder.js

```diff
@@ -3,14 +3,14 @@ const { ContractFactory, Contract } = require('@ethersproject/contracts');
 
 const { ENTRYPOINT, GANACHE_ACCOUNT } = require('../constants');
 const { SMART_CONTRACTS, contractConfiguration } = require('./smart-contracts');
-const GanacheContractAddressRegistry = require('./ganache-contract-address-registry');
+const ContractAddressRegistry = require('./contract-address-registry');
```

Comment:
just a rename, as now it's used both with ganache and anvil - no functionality change